internal/component/pyroscope/scrape/scrape.go

```diff
@@ -21,15 +21,17 @@ import (
 )
 
 const (
-	pprofMemory            string = "memory"
-	pprofBlock             string = "block"
-	pprofGoroutine         string = "goroutine"
-	pprofMutex             string = "mutex"
-	pprofProcessCPU        string = "process_cpu"
-	pprofFgprof            string = "fgprof"
-	pprofGoDeltaProfMemory string = "godeltaprof_memory"
-	pprofGoDeltaProfBlock  string = "godeltaprof_block"
-	pprofGoDeltaProfMutex  string = "godeltaprof_mutex"
+	pprofMemory              string        = "memory"
+	pprofBlock               string        = "block"
+	pprofGoroutine           string        = "goroutine"
+	pprofMutex               string        = "mutex"
+	pprofProcessCPU          string        = "process_cpu"
+	pprofFgprof              string        = "fgprof"
+	pprofGoDeltaProfMemory   string        = "godeltaprof_memory"
+	pprofGoDeltaProfBlock    string        = "godeltaprof_block"
+	pprofGoDeltaProfMutex    string        = "godeltaprof_mutex"
+	defaultScrapeInterval    time.Duration = 15 * time.Second
+	defaultProfilingDuration time.Duration = 14 * time.Second
```

Comment:
We're not using defaultScrapeInterval anywhere.

If we want to provide a sensible default, should we make it like this and wire the defaultScrapeInterval in NewDefaultArguments?

```suggestion
	defaultScrapeInterval    time.Duration = 15 * time.Second
	defaultProfilingDuration time.Duration = defaultScrapeInterval - 1 * time.Second
```